db/models.py

```diff
@@ -50,3 +54,70 @@
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(
+        AUTH_USER_MODEL, on_delete=models.CASCADE, related_name="orders"
```

Comment:
Ensure that `AUTH_USER_MODEL` is correctly referenced as `settings.AUTH_USER_MODEL` if the import statement is corrected.

Reply:
https://stackoverflow.com/questions/54710952/django-importing-auth-user-model